zapi.go

```diff
@@ -33,6 +36,25 @@ func GetDefaultPrinter(buf *uint16, bufN *uint32) (err error) {
 	return
 }
 
+func GetPrinterDriver(name string, buf *byte, bufN uint32) (err error) {
```

Comment:
This file is generated by running "go generate" command. Do not change it manually. This file contains all Windows API, so instead of
```
func GetPrinterDriver(name string, buf *byte, bufN uint32) (err error)
```
you want
```
func GetPrinterDriver(h syscall.Handle, env *uint16, level uint32, di *byte, n uint32, needed *uint32) (err error)
```

So if you add
```
//sys   GetPrinterDriver(h syscall.Handle, env *uint16, level uint32, di *byte, n uint32, needed *uint32) (err error) = winspool.GetPrinterDriver
```
and then run "go generate" command, you will get what you want.

Reply:
I dont know that ! i made the change and generate the code know !